src/hotspot/share/utilities/parse_integer.hpp

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I think you should use the copyright dates from the file where this code got copied from. Same goes for the other new file.